rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1468,146 +1387,63 @@ def _remove_blockchain_accounts_test_start(
     rotki.chain_manager.cache_ttl_secs = 0
     removed_eth_accounts = [ethereum_accounts[0], ethereum_accounts[2]]
     eth_accounts_after_removal = [ethereum_accounts[1], ethereum_accounts[3]]
-    all_eth_balances = ['1000000', '2000000', '3000000', '4000000']
-    token_balances = {A_RDN: ['0', '0', '450000000', '0']}
-    eth_balances_after_removal = ['2000000', '4000000']
-    token_balances_after_removal = {}
-    starting_liabilities = {A_DAI: ['5555555', '1000000', '0', '99999999']}
-    after_liabilities = {A_DAI: ['1000000', '99999999']}
 
-    if query_balances_before_first_modification:
-        # Also test by having balances queried before removing an account
-        setup = setup_balances(
-            rotki,
-            ethereum_accounts=ethereum_accounts,
-            btc_accounts=btc_accounts,
-            eth_balances=all_eth_balances,
-            token_balances=token_balances,
-            liabilities=starting_liabilities,
-        )
-        with ExitStack() as stack:
-            setup.enter_blockchain_patches(stack)
-            response = requests.get(api_url_for(
-                api_server,
-                "blockchainbalancesresource",
-            ))
-    setup = setup_balances(
-        rotki,
-        ethereum_accounts=ethereum_accounts,
-        btc_accounts=btc_accounts,
-        eth_balances=all_eth_balances,
-        token_balances=token_balances,
-        liabilities=starting_liabilities,
-    )
+    # The application has started with 4 ethereum accounts. Remove two of them.
+    response = requests.delete(api_url_for(
+        api_server,
+        "blockchainsaccountsresource",
+        blockchain='ETH',
+    ), json={'accounts': removed_eth_accounts, 'async_query': async_query})
+    if async_query:
+        task_id = assert_ok_async_response(response)
+        wait_for_async_task(api_server, task_id)
+    else:
+        assert_proper_response(response)
 
-    # The application has started with 4 ethereum accounts. Remove two and see that balances match
-    with ExitStack() as stack:
-        setup.enter_ethereum_patches(stack)
-        response = requests.delete(api_url_for(
-            api_server,
-            "blockchainsaccountsresource",
-            blockchain='ETH',
-        ), json={'accounts': removed_eth_accounts, 'async_query': async_query})
-        if async_query:
-            task_id = assert_ok_async_response(response)
-            result = wait_for_async_task_with_result(api_server, task_id)
-        else:
-            result = assert_proper_response_with_result(response)
+    assert rotki.chain_manager.accounts.eth == eth_accounts_after_removal
 
-    assert_eth_balances_result(
```

Comment:
Generally all these. You are deleting very important tests which I don't think are mirrored anywhere else. You should not do that. You need to keep all these and make sure you query the balances after adding the account and checking it's correct.

Only if you can point me to another duplicated tests that tests each functionality of a test whose balance query you removed will it be okay to indeed remove it